src/index.html

```diff
@@ -3,12 +3,82 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="attr-req-value"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="bigcont">
+      <a
+        href="#home"
+        class="link"
+      >
+        <img
+        src="images/logo.png"
+        alt="logo"
+        class="pic"
+      >
+      </a>
+      <nav class="nav-elems">
+        <ul class="container">
+          <li
+            class="color elem first"
+          >
+            <a class="is-active links">
```

Comment:
please change this class name simple so that it can describe itself, somthing like 
```suggestion
            class="nav__link is-active"
```